Last.FM/child.js

```diff
@@ -0,0 +1,22 @@
+function Loader() {}
```

Comment:
looks like `Loader` should be a base class for `XhrLoader` because of their names